### ISSUE TITLE
Document Resource.representations

### ISSUE THREAD

**Issue by nharraud** [flask-restful/flask-restful#521]
After digging in fask-restful code I found `Resource.representations` which does exactly what I want, i.e: having different representations per resource. This is really a great feature but I couldn't find any reference to it in the documentation.
Is it because this is not officially supported and might be removed or is it just missing in the doc?

**Comment by joshfriend** [comment]
I think this is what you were looking for:

http://flask-restful.readthedocs.org/en/0.3.4/api.html#flask_restful.Api.representation
http://flask-restful.readthedocs.org/en/0.3.4/extending.html#content-negotiation

**Comment by joshfriend** [comment]
Whoops, thats not what you were talking about. Is [this the undocumented feature](https://github.com/flask-restful/flask-restful/blob/9bfb9f87215820c970b30e3aaa182adceb3e6bc8/flask_restful/__init__.py#L78)?

**Comment by nharraud** [comment]
No, this one is documented here: http://flask-restful-cn.readthedocs.org/en/latest/extending.html#response-formats. I'm talking about this one: https://github.com/flask-restful/flask-restful/blob/master/flask_restful/__init__.py#L565

**Comment by justanr** [comment]
That's documented a little further down that same page: http://flask-restful-cn.readthedocs.org/en/latest/extending.html#resource-method-decorators

**Comment by nharraud** [comment]
@justanr Sorry I must have selected the wrong line. I meant the line just above:
https://github.com/flask-restful/flask-restful/blob/a191b2879dbeceff0ab5dd4d57dfa7ecb4c630f0/flask_restful/__init__.py#L564

i.e: `Resource.representations`
